src/PAModel/PAConvert/Yaml/YamlConverter.cs

```diff
@@ -0,0 +1,58 @@
+using Microsoft.PowerPlatform.Formulas.Tools.Yaml;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.PowerPlatform
+{
+    /// <summary>
+    /// Convert a dictionary to and from yaml
+    /// </summary>
+    public static class YamlConverter
+    {
+        public static Dictionary<string, string> Read(TextReader reader, string filenameHint = null)
+        {
+            var d = new Dictionary<string, string>(StringComparer.Ordinal);
```

Comment:
very minor nit: this deserves a better name than `d`

Reply:
updated